packages/cli-e2e/utils/login.ts

```diff
@@ -8,14 +10,20 @@ function isLoginPage(page: Page) {
   return page.url().match(/https:\/\/platform.*cloud\.coveo\.com\/login/);
 }
 
-function isLoggedIn(page: Page) {
-  return page.url().match(/http:\/\/127\.0\.0\.1:32111/);
+export async function isLoggedin() {
+  const currentAccount = userInfo().username;
+  const accessToken = await getPassword(
+    'com.coveo.cli.access.token',
+    currentAccount
+  );
+  return accessToken !== null;
 }
 
 function waitForLoginPage(browser: Browser) {
   return new Promise<Page>((resolve) => {
-    browser.on('targetchanged', (page: Page) => {
-      if (page.url && isLoginPage(page) !== null) {
+    browser.on('targetchanged', async (target: Target) => {
+      const page = await target.page();
+      if (page && isLoginPage(page) !== null) {
```

Comment:
I had the wrong type. [targetchanged](https://pptr.dev/#?product=Puppeteer&version=v8.0.0&show=api-event-targetchanged)  returns a [Target](https://pptr.dev/#?product=Puppeteer&version=v8.0.0&show=api-class-target) object... not a page object.